tools/ts_library_builder/build_library.ts

```diff
@@ -227,6 +227,10 @@ export function mergeGlobal({
   // to extract the type and add it to the global variable map
   sourceFile.forEachChild(node => {
     if (TypeGuards.isExpressionStatement(node)) {
+      console.log(
```

Comment:
Do you really want this debug code in the final?  You should leverage the silent option if you really want to output this. 😀

Reply:
😅 yeah, forgot about that